docs/how-to/run-container.md

```
@@ -0,0 +1,14 @@
+# Run in a container
+
+Pre-built containers with edge-containers-cli and its dependencies already
+installed are available on [Github Container Registry](https://ghcr.io/epics-containers/edge-containers-cli).
+
+## Starting the container
+
+To pull the container from github container registry and run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)